setup.cfg

```diff
@@ -38,24 +89,19 @@ bootstrap =
     import pandas as pd
     np  # avoiding error when importing again numpy or pandas
     pd  # (in some cases we want to do it to show users)
-ignore = E203,  # space before : (needed for how black formats slicing)
-         E402,  # module level import not at top of file
-         W503,  # line break before binary operator
-         # Classes/functions in different blocks can generate those errors
-         E302,  # expected 2 blank lines, found 0
-         E305,  # expected 2 blank lines after class or function definition, found 0
-         # We use semicolon at the end to avoid displaying plot objects
-         E703,  # statement ends with a semicolon
-         E711,  # comparison to none should be 'if cond is none:'
-
+ignore =
+    E203,  # space before : (needed for how black formats slicing)
+    E402,  # module level import not at top of file
+    W503,  # line break before binary operator
+    E302,  # expected 2 blank lines, found 0
+    E305,  # expected 2 blank lines after class or function definition, found 0
+    E703,  # statement ends with a semicolon
+    E711,  # comparison to none should be 'if cond is none:'
 exclude =
     doc/source/development/contributing_docstring.rst,
-    # work around issue of undefined variable warnings
-    # https://github.com/pandas-dev/pandas/pull/38837#issuecomment-752884156
```

Comment:
Is there a reason you are removing those comments? (and below some occurrences as well)

Reply:
Maybe we shouldn't use setup-cfg-fmt.

Reply:
If there is no way to configure the formatter to keep the comments, then I think we should indeed not use it

Reply:
Apparently not implemented at the moment, see https://github.com/asottile/setup-cfg-fmt/pull/21#issuecomment-533308788. 
So I would propose to remove the use of setup-cfg-fmt, and add back the comments (can of course keep the other formatting improvements)